sdk-dotnet/LittleHorse.Sdk/CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# LittleHorse Contributing .NET SDK
+
+You need to package the project to generate the nuget file `LittleHorse.{version}.nupkg`
+
+```dotnet pack```
+
+How to set a local nuget directory packages to call its dependencies, you should specify where your .nupkg is created
+
+```dotnet nuget add source {{your_local_path}}/littlehorse/sdk-dotnet/LittleHorse.Sdk/bin/Debug --name LocalNugetPackages```
```

Comment:
Is it possible to use relative paths?
```suggestion
```dotnet nuget add source ./bin/Debug --name LocalNugetPackages```
```

Reply:
I belive yes, lets see

Reply:
done